app/views/reports/regions/who_report.csv.erb

```diff
@@ -0,0 +1,103 @@
+<%
+  csv = CSV.generate(headers: true) do |csv|
+  csv << ["Facility Report #{@period.to_date.strftime("%B %Y")}"]
+
+  # need variable width header here
+
+  # HEADERS
+  month_labels = @months.map { |month| month.value.strftime("%b-%Y") }
+  csv << [
+    "Sr. No",
+    "Name of the District",
+    "Name of the Facility",
+    "Type of Facility",
+    "Name of the Block",
+    "Active/Inactive (Inactive facilities have 0 BP measures taken)",
+    "Est. HTN Population",
+    "Total Reg",
+    "Total Assigned Patients",
+    "Total LTFU",
+    "Died",
+    "Total Patients under Care",
+    month_labels,
+    month_labels,
+    "BP Controlled",
+    "BP not Controlled",
+    "Missed Visit",
+    "Visited but BP not taken",
+    "Amlodipine",
+    "ARBs/ACE Inhibitors",
+    "Diuretic"
+  ].flatten
+
+  # DISTRICT ROW
+  dead = Patient.where(assigned_facility_id: @region.facilities.pluck(:id), status: "dead").count
+  registered_by_month = @months.map do |month|
+    @dashboard_analytics.sum { |_, data| data.dig(:registered_patients_by_period, month.value) || 0 }
+  end
+  follow_up_by_month = @months.map do |month|
+    @dashboard_analytics.sum { |_, data| data.dig(:follow_up_patients_by_period, month.value) || 0 }
```

Comment:
I'm working on adding follow up numbers to the Repo in https://github.com/simpledotorg/simple-server/pull/2436, so hopefully these can use the repo too soon.